### DISCUSSION TITLE
Constantly getting this error in a discord bot using serenity

### DISCUSSION THREAD

**Discussion by Milo123459** [SeaQL/FireDBG.for.Rust#7] [Bug]
```rs
ERROR interaction_create{cmd="/singleplayer" user=450212014912962560}: sushibot::commands::singleplayer: Cannot respond to slash command: Unknown interaction
thread 'main' panicked at debugger/src/debugger.rs:660:13:
assertion failed: !active_frames.is_empty()
```
I'm using the FireDBG for VSCode, and I have also tried the extension. I'm seeing this consistently happen.

**Comment by tyt2y3** [comment]
Interesting. I have never seen this before. What does your program look like? Do you have a reproducible example?

**Reply by Milo123459** [reply]
Not currently, but it is using Tokio 1.35.0 - what other information would be useful?

**Reply by tyt2y3** [reply]
Oh, then I get it now. Async is still a WIP. We will get to there, hopefully soon!

**Reply by Milo123459** [reply]
Awesome! Is there anyway I can help out?
